commands/publish/command.js

```diff
@@ -68,6 +68,13 @@ exports.builder = yargs => {
       type: "string",
       requiresArg: true,
     },
+    "otp-rate-limit-delay": {
```

Comment:
I tried to find a better name but I must confess I was at a loss here.

Reply:
It's an awkward concept to express, I think you've done well enough. :)